dvc/stage.py

```diff
@@ -982,18 +982,24 @@ def checkout(
         relink=False,
         filter_info=None,
     ):
-        failed_checkouts = []
+        checkouts = {"failed": [], "added": [], "modified": []}
         for out in self._filter_outs(filter_info):
-            failed = out.checkout(
-                force=force,
-                tag=self.tag,
-                progress_callback=progress_callback,
-                relink=relink,
-                filter_info=filter_info,
-            )
-            if failed:
-                failed_checkouts.append(failed)
-        return failed_checkouts
+            try:
+                is_modified = out.checkout(
```

Comment:
What do you think to apply one naming, either `is_modified` or `modified everywhere?

Reply:
Thanks for the suggestion. I have fixed it.